src/Menu.jsx

```diff
@@ -75,7 +76,8 @@ const Modes = {
   DISABLED: 0,
   VIEW: 1,
   ANNOTATE: 2,
-  BUILD_CLASSIFIER: 3
+  BUILD_CLASSIFIER: 3,
+  PREDICT: 4
```

Comment:
this will conflict with the #23 work, but if you want to land this ahead of that I can take the merge pain.

Reply:
ok, thanks!